azure-devops/azext_devops/dev/repos/arguments.py

```diff
@@ -35,9 +33,9 @@ def load_code_arguments(self, _):
                          'exact match on the --branch argument. ' +
                          'If value is \'prefix\' the policy is applied across all branch folders that ' +
                          'match the prefix provided by the --branch argument.')
-        context.argument('is_blocking', arg_type=get_enum_type(_TRUE_FALSE_SWITCH),
+        context.argument('is_blocking', arg_type=get_three_state_flag(),
```

Comment:
Are we not changing the param names is_blocking -- blocking?

Reply:
is_blocking is member of PolicyConfiguration and a property, so I am not sure what is the reason behind changing it's name just to create more confusion for users

Reply:
cc: @snankani  @geverghe : let me know if we should change --is-blocking to --blocking

Reply:
IMO it should change to --blocking provided this is adhering to the registering flag (https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags) paradigm listed here. The flag is named as bool_prop_enabled so that it makes sense without having to provide the actual value of true which is assumed here.

With adherence to the registering flags paradigm, now users need not provide the default value. Therefore, the command should make sense with the absence of an explicit value. Therefore here:
1. <With rename> az repos policy --blocking
2. <without rename> az repos policy --is-blocking
In the first, blocking gives clarity about the action that is being taken. Is blocking is still ambiguous about the action being taken - is it blocking? Is not blocking?

Reply:
--blocking is more precise compared to --is-blocking in this case.

Reply:
will update to --blocking and --enabled but just putting my concerns
1. This is already rolled out and being changed without any feedback.. so any existing script using this will be broken
2. If I have to use it in any script then I will have to deal with 2 things now 'blocking' and 'is-blocking' because 'is-blocking' is part of the response from service.